registry/quilt_server/views.py

```diff
@@ -661,9 +661,14 @@ def download_object_preview(owner, obj_hash):
         )
 
 @app.route('/api/package/<owner>/<package_name>/<package_hash>', methods=['PUT'])
+@app.route('/api/package_update/<owner>/<package_name>/<path:package_path>', methods=['POST'])
```

Comment:
(I don't actually know what the path should be. Also, I'm guessing it should to be a `POST`, not a `PUT`, since it's not creating a new entity?)